README.md

```diff
@@ -81,7 +81,7 @@ This build mode assumes you already have compiled LLVM and Clang libraries on yo
       iwyu/build$ cmake -G "Unix Makefiles" -DIWYU_LLVM_ROOT_PATH=/usr/lib/llvm-6.0 ../include-what-you-use
 
       # For IWYU 0.11/Clang 7 and later
-      iwyu/build$ cmake -G "Unix Makefiles" -DCMAKE_PREFIX_PATH=/usr/lib/llvm-7 ../include-what-you-use
+      iwyu/build$ cmake .. -G "Unix Makefiles" -DCMAKE_PREFIX_PATH=/usr/lib/llvm-7
```

Comment:
This won't work if you've followed the checkout instructions before. And it's also inconsistent with the `# For IWYU 0.10...` section above.